crates/bevy_math/src/cubic_splines.rs

```diff
@@ -270,22 +281,286 @@ impl<P: Point> CubicGenerator<P> for CubicBSpline<P> {
     #[inline]
     fn to_curve(&self) -> CubicCurve<P> {
         let char_matrix = [
```

Comment:
This (and the original) should have a comment, either explaining or linking to the mathematical reference so it can be verified.

Reply:
Sorry, I don't quite understand what particularly you want the comment to clarify?